samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -79,13 +79,13 @@ void main(void)
 	printk("Reading some secure memory that NS is allowed to read\n");
 
 	NRF_TIMER1_NS->TASKS_START = 1;
-	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.PART);
+	part = secure_read_word((intptr_t)&NRF_FICR_S->INFO.RAM);
```

Comment:
@joerchan Should not part variable be renamed to info_ram or removed, and printk() below adapted accordingly?

Reply:
It should, @desowin can you amend the commit?

Reply:
fixed